configs/pie-components-config/playwright-lit-visual-config.js

```diff
@@ -51,7 +51,7 @@ export function getPlaywrightVisualConfig () {
                 name: 'visual:mobile',
                 grep: /@mobile/,
                 use: {
-                    ...devices['Pixel 5'],
+                    ...devices['iPhone 11 Pro'],
```

Comment:
Pixel 5 is a 393px width. iPhone 11 Pro is 375px, which matches our `percyWidths` breakpoint.

There was a bit of a bug where the element wasn't properly resizing when Percy set its page to 375px wide, so it's best to align these as close as possible so that we get accurate visual tests.

There are one or two other tests in the docs site that use the `@mobile` flag which this in theory could affect, but I'm not expecting to see a difference.

Reply:
Docs uses the config defined at the root of the repo. The config you've changed is specific to Lit components, so shouldn't have an impact 👍🏼 - https://github.com/justeattakeaway/pie/blob/main/playwright-visual.config.ts#L65

Reply:
On a separate note, I've noticed that `@justeattakeaway/pie-components-config` isn't listed as a devDep in all components, meaning visual tests probably won't trigger for all components for this change. This PR is pretty low risk, but it's something we definitely need to sort.